travharv/config_build.py

```diff
@@ -267,7 +266,16 @@ def build_from_config(self, config_name=str):
         """
         config_file = str(Path.cwd() / self.config_files_folder / config_name)
         dict_object = self._load_yml_to_dict(config_file)
-        return self._makeTravHarvConfigPartFromDict(dict_object, config_name)
+
+        relative_name_config = relative_pathname(
+            Path(config_file), Path.cwd() / self.config_files_folder
```

Comment:
always having odd feelings with explicit occurances of `Path.cwd()` unclear why it is used here?